app/routes/events/components/Event.css

```diff
@@ -1,56 +1,27 @@
 @import url('~app/styles/variables.css');
 
-.feedback {
-  margin-top: 1rem;
+.form {
+  gap: 1rem;
 }
 
 .feedbackText {
   min-height: 40px;
-  min-width: 200px;
+  min-width: 190px;
   max-width: 893px;
   max-height: 300px;
   resize: vertical;
 }
 
-.registeredThumbnails {
-  flex-wrap: wrap;
-  max-height: 130px;
-  overflow: hidden;
-}
-
-.meta {
-  flex: 1;
-  padding: 20px;
-  padding-top: 0;
-  border-left: 1px solid var(--border-gray);
-  margin-left: 3em;
-
-  @media (--small-viewport) {
-    border-left: none;
-    margin-left: 0;
-    padding: 0;
-  }
+.feedbackButton {
+  margin-top: 23.4px; /* The height of the field label */
```

Comment:
Is there no better way to do this?

Reply:
hmm I can't think of a way other than updating the actual field component or moving the label outside - which at the time seemed excessive for this "edge case".

I totally agree that this is hacky and very bad practice - so it's a good thing you point it out!

Reply:
When I come to think of it - moving the label is probably cleaner. I'll look into it!

Reply:
Ended up moving the label outside of the Field component - worked like a charm!